packages/os/os.spec

```diff
@@ -227,6 +237,13 @@ Summary: Setting generator for populating admin container user-data from IMDS.
 %{summary}.
 %endif
 
+%if %{_is_vmware_variant} || %{_is_dev_variant}
+%package -n %{_cross_os}prairie-dog
+Summary: Tools for kdump support
+%description -n %{_cross_os}prairie-dog
+%{summary}.
```

Comment:
`prairiedog` should depend on `kexec-tools` and `makedumpfile`, but we don't really want to delay the `os` package build waiting for those.

I would give it the same treatment we've given `host-ctr`, where we have "Requires" in the spec, and add comments to `Cargo.toml` saying how we'll pull it in via `release` instead. Then just add these tools unconditionally to `release`.

That ties into my other feedback - kdump support is really a distro feature that we'll want an easy way to turn on or off by just setting the right command line parameter in the variant definition.